PrebidMobile/PrebidMobile-rendering/src/main/java/org/prebid/mobile/rendering/bidding/display/MediationBaseAdUnit.java

```diff
@@ -151,6 +150,10 @@ public String getPbAdSlot() {
         return mAdUnitConfig.getPbAdSlot();
     }
 
+    public void setContentObject(ContentObject content) {
```

Comment:
Need to rename this method in the same way as in the PR for the original SDK - `addContent `.